### ISSUE TITLE
THIS IS NOT A ISSUE, I'm writing this to thank you 

### ISSUE THREAD

**Issue by ggperes2019** [Roberto-Nessy/OoT3D_Standalone_Free_Cam#1]
The mod is working on my Citra pretty great, i want to thank you and say to you that you should make a video about the mod(is a suggestion, of course) showing how to install in Citra, so people can have more acess to the mod, because so far the option more popular for making the camera move in 360 on the game is the pc port/Ship of Harkinian. That is it, thanks for reading  : )

**Comment by Geckostya** [comment]
The mod is amazing!
I found how to install it on the Android's citra

You need to create folders 'load/mods/TitleId". To find TitleId you can enable custom textures, then Citra will create "load/textures/TitleId". It is also the number from the ReadMe, 0004000000033600 in my case
Then you need to put files in this folder

Found it there:
https://citra-emu.org/help/feature/game-modding/
https://www.reddit.com/r/EmulationOnAndroid/comments/inzak1/reminder_you_can_also_apply_mods_on_androids/

**Comment by ggperes2019** [comment]
> The mod is amazing! I found how to install it on the Android's citra
> 
> You need to create folders 'load/mods/TitleId". To find TitleId you can enable custom textures, then Citra will create "load/textures/TitleId". It is also the number from the ReadMe, 0004000000033600 in my case Then you need to put files in this folder
> 
> Found it there: https://citra-emu.org/help/feature/game-modding/ https://www.reddit.com/r/EmulationOnAndroid/comments/inzak1/reminder_you_can_also_apply_mods_on_androids/

True, I found a similar mod on Henriko Magnifico website but the mod didn't work for me, so I tried this one and it is awesome. I hope, someone makes a video one day spreading the information about it, I could make the video, but I'm very busy with college and usually when I have free time I spend studying or playing video-games , lol.

P.S - Sorry my English, Portuguese is my primary language because I'm Brazilian, so my level is not very good, thanks for reading.
